src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
The Lottery shouldn't extend ColorSupplier. We can simply check it. 
Can Lottery be a ColorSupplier? The answer is no. So, there is no way for inheritance.
Can Lottery have a ColorSupplier? Yes, definitely. So, ColorSupplier should be a field in the Lottery class.

Read more about it here: https://www.baeldung.com/java-inheritance-composition